core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1594,10 +1313,6 @@ object TestUtils extends Logging {
     }
   }
 
-  def createIsrChangeListener(): MockAlterPartitionListener = {
```

Comment:
`MockAlterPartitionListener` is used by `AbstractPartitionTest` only, so could you move it to `AbstractPartitionTest`?